shenyu-admin/src/main/java/org/apache/shenyu/admin/service/RuleService.java

```diff
@@ -48,6 +53,17 @@ public interface RuleService extends PageService<RuleQueryCondition, RuleVO> {
      * @return rows int
      */
     default int createOrUpdate(final RuleDTO ruleDTO) {
+        // check rule uri condition
+        try {
+            final List<RuleConditionDTO> ruleConditions = ruleDTO.getRuleConditions();
+            List<String> uriPaths = ruleConditions.stream()
+                    .filter(conditionData -> ParamTypeEnum.URI.getName().equals(conditionData.getParamType()))
+                    .map(RuleConditionDTO::getParamValue)
+                    .collect(Collectors.toList());
```

Comment:
`.forEach(PathPatternParser.defaultInstance::parse);`
`collect()` is not needed.